tests/integration/smoke_test.go

```diff
@@ -86,7 +86,8 @@ func (suite *SmokeSuite) TearDownSuite() {
 }
 
 func (suite *SmokeSuite) TestBlockStorage_SmokeTest() {
-	runBlockE2ETest(suite.helper, suite.k8sh, suite.Suite, suite.namespace)
+	runBlockE2ETest(suite.helper, suite.k8sh, suite.Suite, suite.namespace, "Delete")
```

Comment:
Instead of running the entire test again with a different policy, what if you created another pod inside the same test with the different policy? The tests are rather time consuming and we are looking for ways to minimize the time to run them.